electron/launcher/src/components/Login/Login.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { userActions } from '../../_actions';
+import FormBuilder from '../FormBuilder';
+
```

Comment:
relative pathing, is it possible to do pathing from project root/src?

Reply:
done